src/utils/usageReportConsent.js

```diff
@@ -7,3 +7,7 @@ export const getConsentStatus = () => {
 export const setConsentStatus = consent => {
   localStorage.setItem(KEY, !!consent)
 }
+
+export const hasConsentBeenAnswered = () => {
```

Comment:
The logic is reversed between the name and implementation. If the consent has been answered, then it is not null.

Reply:
You know... I actually thought of this too, gotta commit to it what I pick up stuff like that